package.json

```diff
@@ -300,31 +300,31 @@
   },
   "dependencies": {
     "archiver": "^2.1.1",
-    "cs-jsforce-metadata-tools": "^1.2.10",
     "elegant-spinner": "^1.0.1",
     "fs-extra": "^5.0.0",
-    "globule": "^1.2.0",
-    "jsforce": "github:celador/jsforce",
```

Comment:
Is there a reason to remove this?  There are methods used in the extension from this fork.  If necessary to have recent updates from the upstream please let me know and I will update from the upstream.